src/approaches/pg3_approach.py

```diff
@@ -23,10 +23,11 @@
 from typing_extensions import TypeAlias
```

Comment:
is the high-level idea of this PR that we want to use the LDL policies learned by PG3 to guide sesame rather than what we had before, which was to use it to guide `run_astar`?

Reply:
so what would happen if we were to just use regular PG3 on an environment that requires bilevel planning? does that even make any sense at all?

Reply:
Exactly, we use the ldl policies to guide sesame. Initially, PG3 couldn't be used in TAMP problems, so we added low-level option search to it. That worked well for simple TAMP problems, but it would get stuck in environments like cover. To overcome this, we now use the ldl policy pg3 generates to guide bilevel planning.